python/taichi/main.py

```diff
@@ -56,13 +56,6 @@ def __init__(self, test_mode: bool = False):
 
         print(self._get_friend_links())
 
-        if 'TI_DEBUG' in os.environ:
-            val = os.environ['TI_DEBUG']
-            if val not in ['0', '1']:
-                raise ValueError(
-                    "Environment variable TI_DEBUG can only have value 0 or 1."
-                )
-
```

Comment:
A little curious: Do we want to deprecate `TI_DEBUG` now?It is still hinted in the [doc](https://taichi.readthedocs.io/en/stable/global_settings.html?highlight=TI_DEBUG#specifying-ti-init-arguments-from-environment-variables)

Reply:
It's done in `ti.init` now, the detection here is duplicate.